### PR TITLE
Statement tasks only look at tagpools with unused tags

### PR DESCRIPTION
**Pull request by justinvdm**
The task should look at all known tagpools.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

File: go/vumitools/api.py [hunk 1]
```diff
@@ -237,9 +244,12 @@ def tagpools(self):
             if free_tags:
                 available_pools.append(pool)
 
-        pool_data = dict([(pool, (yield self.api.tpm.get_metadata(pool)))
-                          for pool in available_pools])
-        returnValue(TagpoolSet(pool_data))
+        returnValue((yield self._tagpool_set(available_pools)))
+
+    @Manager.calls_manager
+    def known_tagpools(self):
```
Comment (hodgestar) [inline go/vumitools/api.py]: Should this be on `VumiApi` rather than `VumiUserApi`?

**Comment by hodgestar** [comment]
:+1:

**Comment by hodgestar** [comment]
And thank you for adding all the tests for TagpoolSet -- that was my code originally and I should have added tests for it when I wrote it. :/

**Pull request merged**